### PR TITLE
regression fix: missing priority icon in torrent cell

### LINKED ISSUE #5826: MacOS torrent priority chevron no longer visible
### What is the issue?

I just noticed the up or down chevron that denotes torrent priority is no longer visible in the macOS client.

An assumption, given that it was the last major change to the macOS client, but possibly related to #5147. 

### Which application of Transmission?

macOS app

### Which version of Transmission?

4.1.0-dev (22da17f13f)

### PR DESCRIPTION
**Pull request by sweetppro**
Fix #5826 (regression from #5147) - autolayout alternative to #5846

@couer, here's an alternative to your proposed fix (#5846), using auto layout...
I haven't added in your refactored TorrentCell and SmallTorrentCell - I figured you could maybe submit another PR for that (assuming this version is preferable to your solution)

### CODE REVIEW

**Comment by Coeur** [comment]
@ckerr once verified, we'll merge both: first the present one by sweetppro, then mine for the refactoring to avoid duplicate code. I'll test it today.

**Comment by ckerr** [comment]
OK. I'll wait for y'alls go-ahead before merging then, thanks for the info!

**Review by Coeur — APPROVED** [review]
@ckerr tested and code-verified: good to merge

**Pull request merged**
